build.gradle

```diff
@@ -1,121 +1,19 @@
-}
-
-repositories {
-    mavenLocal()
-    mavenCentral()
-}
-
-def isNonStable = { String version ->
-    def stableKeyword = ['RELEASE', 'FINAL', 'GA'].any { it -> version.toUpperCase().contains(it) }
-    def regex = /^[0-9,.v-]+(-r)?$/
-    return !stableKeyword && !(version ==~ regex)
-}
-
-dependencyUpdates {
-    rejectVersionIf {
-        isNonStable(it.candidate.version) && !isNonStable(it.currentVersion)
-    }
-}
-
-configurations {
-    agentDeps
-    agentDeps.transitive = false
-}
-
-compileKotlin {
-    kotlinOptions {
-        jvmTarget = '11'
-    }
-}
-compileTestKotlin {
-    kotlinOptions {
-        jvmTarget = '11'
-    }
-}
-
-ext {
-    ext {
-        drivers = [
-                'chromeHeadless'
-        ]
-    }
-}
-
-dependencyCheck {
-    failBuildOnCVSS = 5
-    suppressionFiles = ['dependency-check-suppress-spring.xml']
-    format = 'ALL'
-    analyzers {
-        assemblyEnabled = false
-    }
-}
-
-project.sourceCompatibility = 11
-project.targetCompatibility = 11
-
-group = 'uk.gov.justice.digital.hmpps'
-
-def today = Instant.now()
-def todaysDate = LocalDate.now().format('yyyy-MM-dd')
-version = System.getenv('BUILD_NUMBER') ?: todaysDate
-
-bootJar {
-    manifest {
-        attributes 'Implementation-Title': rootProject.name, 'Implementation-Version': rootProject.version
-    }
-}
-
-springBoot {
-    buildInfo {
-        properties {
-            artifact = rootProject.name
-            version = version
-            group = group
-            name = rootProject.name
-            time = today
-            additional = [
-                    by                   : System.properties['user.name'],
-                    operatingSystem      : "${System.properties['os.name']} (${System.properties['os.version']})",
-                    machine              : InetAddress.localHost.hostName
-            ]
-        }
-    }
-}
-
-dependencyManagement {
-    imports {
-        mavenBom(SpringBootPlugin.BOM_COORDINATES)
-    }
+    id "uk.gov.justice.hmpps.gradle-spring-boot" version "0.1.0"
 }
 
 configurations {
     compile.exclude  module: "tomcat-jdbc"
 }
 
-ext ['spring-security.version'] = '5.3.0.RELEASE' // Updated since spring-boot-starter-oauth2-resource-server-2.2.5.RELEASE only pulls in 5.2.2.RELEASE (still affected by CVE-2018-1258 though)
+ext ['spring-security.version'] = '5.3.1.RELEASE' // Updated since spring-boot-starter-oauth2-resource-server-2.2.5.RELEASE only pulls in 5.2.2.RELEASE (still affected by CVE-2018-1258 though)
 
 dependencies {
-    annotationProcessor "org.springframework.boot:spring-boot-configuration-processor"
+    annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
     annotationProcessor 'org.projectlombok:lombok:1.18.12'
+
     testAnnotationProcessor 'org.projectlombok:lombok:1.18.12'
 
-    implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation 'org.springframework.boot:spring-boot-devtools'
```

Comment:
This isn't in the plugin... not required?

Reply:
not sure if it was a general copy and paste from a different project anyway.  Didn't think that devtools did anything if it didn't have a ui?